__tests__/files.service.test.ts

```diff
@@ -83,7 +88,9 @@ describe('File Service', () => {
 
   it('#getFileContent should read file content with utf8 encoding', () => {
     const path = 'file.json';
-    const readFileSyncSpy = jest.spyOn(fs, 'readFileSync').mockImplementation();
+    // const readFileSyncSpy = jest.spyOn(fs, 'readFileSync').mockImplementation();
+    const readFileSyncSpy = jest.fn();
+    jest.unstable_mockModule('fs', () => ({ readFileSync: readFileSyncSpy }));
```

Comment:
This mock does not work. When executing the test, it throws the error: "_ENOENT: no such file or directory, open 'file.json'_"